src/Visualization.jl

```diff
@@ -8,15 +8,14 @@ function Makie.plot(tn::TensorNetwork{A}; kwargs...) where {A}
     f = Makie.Figure()
 
     p, ax = Makie.plot!(f[1, 1], tn; kwargs...)
-    # display(f)
+    display(f)
 
-    # return f, ax, p
-    return f
+    return f, ax, p
 end
 
 function Makie.plot!(f::Makie.GridPosition, tn::TensorNetwork{A}; labels = false, kwargs...) where {A}
-    # scene = Makie.Scene()
-    # default_attrs = Makie.default_theme(scene, GraphPlot)
+    scene = Makie.Scene()
+    default_attrs = Makie.default_theme(scene, GraphPlot)
```

Comment:
These variables are not used.
```suggestion
    # scene = Makie.Scene()
    # default_attrs = Makie.default_theme(scene, GraphPlot)
```

Actually, we could delete them.